src/main/webapp/scripts/my-trips.js

```diff
@@ -373,28 +477,85 @@ async function saveTrip(hotelID, hotelRef, hotelName) {
     });
   }
 
+  const tripTitle = document.getElementById('trip-title').value;
   const requestBody = {
-    title: document.getElementById('trip-title').value,
+    title: tripTitle,
     hotel_id: hotelID,
     hotel_img: hotelRef,
     hotel_name: hotelName,
     rating: -1,
     locations: locationData,
   };
+  console.log(timestamp);
```

Comment:
nit: remove console.log() when done debugging

Reply:
Right, forgot to remove this before making the PR. Thanks for pointing it out!